Lib/ast.py

```diff
@@ -1264,12 +1265,7 @@ def visit_Subscript(self, node):
             if (isinstance(node.slice, Index)
                     and isinstance(node.slice.value, Tuple)
                     and node.slice.value.elts):
-                if len(node.slice.value.elts) == 1:
-                    elt = node.slice.value.elts[0]
-                    self.traverse(elt)
-                    self.write(",")
-                else:
-                    self.interleave(lambda: self.write(", "), self.traverse, node.slice.value.elts)
+                self.items_view(lambda: self.write(", "), node.slice.value.elts)
```

Comment:
This seems incorrect: `items_view` will call the lambda with one argument:

```
            traverser(items[0])
```

Reply:
ups, my bad. Thank you for noticing it